sensor/kubernetes/listener/resources/secrets.go

```diff
@@ -284,6 +288,11 @@ func (s *secretDispatcher) processDockerConfigEvent(secret, oldSecret *v1.Secret
 						ImageIntegration: ii,
 					},
 				})
+				if action == central.ResourceAction_REMOVE_RESOURCE {
```

Comment:
It looks odd that this event is generated both at the deployments and secrets handler.

Reply:
And `REMOVE` events are only handled if `ourcedAutogeneratedIntegrations` is enabled.